app/javascript/components/pxe-customization-template-form/pxe-customization-template-form.schema.js

```diff
@@ -3,67 +3,73 @@ import { componentTypes, validatorTypes } from '@@ddf';
 const setFormat = (type) => (typeof type !== 'undefined' && type === 'CustomizationTemplateKickstart' ? 'shell' : 'xml');
 
 const createSchema = ({ type } = {}) => ({
-  fields: [
-    {
-      component: componentTypes.TEXT_FIELD,
-      name: 'name',
-      id: 'name',
-      label: __('Name'),
-      maxLength: 255,
-      isRequired: true,
-      validate: [{ type: validatorTypes.REQUIRED }],
-    },
-    {
-      component: componentTypes.TEXT_FIELD,
-      name: 'description',
-      id: 'description',
-      label: __('Description'),
-      maxLength: 100,
-    },
-    {
-      component: componentTypes.SELECT,
-      name: 'pxe_image_type_id',
-      id: 'pxe_image_type_id',
-      label: __('Image Type'),
-      placeholder: __('<Choose>'),
-      includeEmpty: true,
-      loadOptions: () => API.get(`/api/pxe_image_types?attributes=name,id&expand=resources`).then(({ resources }) =>
-        resources.map(({ id, name }) => ({ value: id, label: name }))),
-      isRequired: true,
-      validate: [{ type: validatorTypes.REQUIRED }],
-    },
-    {
-      component: componentTypes.SELECT,
-      name: 'type',
-      id: 'type',
-      label: __('Type'),
-      placeholder: __('<Choose>'),
-      includeEmpty: true,
-      options: [
-        {
-          label: 'Kickstart',
-          value: 'CustomizationTemplateKickstart',
-        },
-        {
-          label: 'Sysprep',
-          value: 'CustomizationTemplateSysprep',
-        },
-        {
-          label: 'CloudInit',
-          value: 'CustomizationTemplateCloudInit',
-        },
-      ],
-      isRequired: true,
-      validate: [{ type: validatorTypes.REQUIRED }],
-    },
-    {
-      component: 'code-editor',
-      id: 'script',
-      name: 'script',
-      label: __('Script'),
-      mode: setFormat(type),
-    },
-  ],
+  fields: [{
```

Comment:
It is showing lot of changes. Is something changed in this file?

Reply:
I just added the subform to all of the PXE react forms so that their all consistently the same and also because they were there in the original forms